README.md

```diff
@@ -10,13 +10,15 @@ See [this page](https://docs.docker.com/engine/getstarted/step_one/) for more in
 
```

Comment:
(Huh, you can't comment on non-modified parts of the doc)

Change "These three flavors" to "These _four_ flavors" above